service/defaultconfig/config.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package defaultconfig
+
+import (
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/config/configmodels"
+)
+
+// CreateDefaultConfig creates default configuration.
+func CreateDefaultConfig(factories config.Factories) *configmodels.Config {
+	otlpRec := factories.Receivers["otlp"].CreateDefaultConfig()
+	batch := factories.Processors["batch"].CreateDefaultConfig()
+	queuedRetry := factories.Processors["queued_retry"].CreateDefaultConfig()
+	return &configmodels.Config{
+		Receivers:  configmodels.Receivers{otlpRec.Name(): otlpRec},
+		Processors: configmodels.Processors{batch.Name(): batch, queuedRetry.Name(): queuedRetry},
```

Comment:
We intend to declare "queue_retry" obsolete, since the exporters now are supposed to have separate queues. I think we don't need "queue_retry" in the default config anymore.
cc @bogdandrutu